apps/coscrad-frontend/src/components/resources/spatial-features/spatial-feature-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,73 @@
+import {
+    GeometricFeatureType,
+    ICategorizableDetailQueryResult,
+    ISpatialFeatureViewModel,
+} from '@coscrad/api-interfaces';
+import { isNullOrUndefined } from '@coscrad/validation-constraints';
+import { Grid, styled } from '@mui/material';
+import { SinglePropertyPresenter } from '../../../utils/generic-components';
+import { ResourceNamePresenter } from '../../../utils/generic-components/presenters/resource-name-presenter';
+import { CoscradMainContentContainer } from '../../../utils/generic-components/style-components/coscrad-main-content-container';
+import { FunctionalComponent } from '../../../utils/types/functional-component';
+import { LineTextPresenter } from './thumbnail-presenters/line-text-presenter';
+import { PointTextPresenter } from './thumbnail-presenters/point-text-presenter';
+import { PolygonTextPresenter } from './thumbnail-presenters/polygon-text-presenter';
+
+interface HasCoordinates<T = unknown> {
+    coordinates: T;
+}
+
+const lookupTable: { [K in GeometricFeatureType]: FunctionalComponent<HasCoordinates> } = {
+    [GeometricFeatureType.point]: PointTextPresenter,
+    [GeometricFeatureType.line]: LineTextPresenter,
+    [GeometricFeatureType.polygon]: PolygonTextPresenter,
+};
+
+const StyledPlaceIcon = styled('img')({
+    width: '150px',
+});
+
+export const SpatialFeatureDetailFullViewPresenter = (
```

Comment:
I'm into having an integrated full-screen view for the map with panels that come up from the bottom with further details on a place with the self-notes as well as the side panel for connected resources.  This is just for the time being since there's a link to view the place from the pop-up.